app/assets/stylesheets/wiki.css

```diff
@@ -68,9 +68,9 @@ a.navbar-offset {
     top: -100px;
     visibility: hidden;
 }
+
 @media (max-width: 768px) {
-  .btn.btn-primary {
+  .add {
```

Comment:
Here this is good but also we should try to make it even more unique, since `add` may be a common classname. Could it be called `add-subscriptions`?

Reply:
changed class name to `add-subscriptions` to make it more unique.
please review
thank you